### PR TITLE
Boulder CZO finished

### PR DESCRIPTION
**Pull request by spencerrhea**
@vlahm

### CODE REVIEW

File: src/global/global_helpers.R [hunk 1]
```diff
@@ -3846,6 +3845,10 @@ convert_to_gl <- function(x, input_unit, molecule) {
         return(x)
     }
 
+    if(grepl('ppm', input_unit)){
```
Comment (vlahm) [inline src/global/global_helpers.R:3848]: derelict chunk?

**Review by vlahm — CHANGES_REQUESTED** [review]
@spencerrhea There's one chunk that would always throw an error if it ran. give it a chez plz

**Comment by vlahm** [comment]
@spencerrhea just a reminder. there's a chunk here, within convert_to_gl, that looks like it's still under development.

File: src/global/global_helpers.R (hunk 1 above)
Comment (spencerrhea) [reply src/global/global_helpers.R]: @vlahm Great catch, thanks! I was going to build that but found it not necessary.

**Review by vlahm — APPROVED** [review]

**Pull request merged**